website/docs/basics/glossary.md

```diff
@@ -2331,6 +2617,33 @@ Uncles are paid a reduced block reward when they are found and reported.
 
 On Nervos, Uncles are tracked by consensus to adjust the block interval of the network.
 
+An uncle block has to meet the following conditions:
+
+- An uncle should not be on the main chain; in other words, it should not be an uncle if it includes any block from the main chain.
+- Uncle’s block number must be smaller than the block‘s number that later includes it.
+- Uncle‘s parent must be on the main chain, or uncle’s parent must also be an uncle. In other words, uncle must be linkable to the main chain in any way. It can never be a random block that is not on the main chain.
+
+```
+- if !snapshot.is_main_chain(&uncle.hash()) // It should not be on the main chain.
+- && !snapshot.is_uncle(&uncle.hash()) // It should not be an uncle twice. 
+- && uncle.number() < candidate_number // The block number of the uncle should be smaller than that of any block on the main chain. 
+- && (uncles.iter().any(|u| u.hash() == parent_hash) 
+|| snapshot.is_main_chain(&parent_hash) // Uncle block should not be parentless. The parent of un uncle must be a block on the main chain or another uncle. 
+|| snapshot.is_uncle(&parent_hash))
+```
+
+<img src={useBaseUrl("img/uncle_rule.png")}/>
+
```

Comment:
![image](https://user-images.githubusercontent.com/46400566/232360054-53cc4073-6b77-484c-8338-5454bd156e75.png)

Where is `image/uncle_rune.png` ? I think this file doesn't tracked by git.